bot/cogs/snakes.py

```diff
@@ -2,10 +2,33 @@
 import logging
 from typing import Any, Dict
 
+from googleapiclient.discovery import build
 from discord.ext.commands import AutoShardedBot, Context, command
+from discord import Embed
+
+import aiohttp
+import json
+import async_timeout
+import random
+import difflib
 
 log = logging.getLogger(__name__)
 
+# Probably should move these somewhere
+BASEURL = "https://en.wikipedia.org/w/api.php?format=json&action=query&prop=extracts|pageimages&exintro=&explaintext=&titles={}&redirects=1"
```

Comment:
this line is 140 characters. it's not gonna pass linting and it's just not okay to do even if it did.